mews-api.md

```diff
@@ -972,8 +972,10 @@ There are certain rules that need to be followed in order for Mews to process th
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
 | `code` | `string` | required \(always\) | Unique code of the reservation within the whole booking. Any value but `_` accepted. No characters limit. |
-| `spaceTypeCode` | `string` | required \(exc. Cancellation\) | Space type code of the reservation. |
-| `ratePlanCode` | `string` | required \(exc. Cancellation\) | Rate type code of the reservation. |
+| `spaceTypeCode` | `string` | required \(exc. Cancellation\) | Space type code of the reservation. 
```

Comment:
Its wrongly formated. Its on new line so it does new column. Click on 3 dots and view fiel you will see how documentation will looks like

Reply:
Done